server/notification-providers/mattermost.js

```diff
@@ -20,7 +20,7 @@ class Mattermost extends NotificationProvider {
                 return okMsg;
             }
 
-            const mattermostChannel = notification.mattermostchannel;
+            const mattermostChannel = notification.mattermostchannel.toLowerCase;
```

Comment:
I believed that it should be .toLowerCase() ?

Reply:
Yes that is correct my bad, fixed the same.